redisinsight/ui/src/components/monitor/Monitor/Monitor.tsx

```diff
@@ -1,18 +1,20 @@
-import React from 'react'
+import React, { useState } from 'react'
 import cx from 'classnames'
 import {
   EuiButtonIcon,
   EuiFlexGroup,
   EuiFlexItem,
   EuiIcon,
+  EuiSwitch,
   EuiTextColor,
   EuiToolTip,
 } from '@elastic/eui'
-import { AutoSizer } from 'react-virtualized'
+import AutoSizer from 'react-virtualized-auto-sizer'
 
 import { IMonitorDataPayload } from 'uiSrc/slices/interfaces'
 import { ReactComponent as BanIcon } from 'uiSrc/assets/img/monitor/ban.svg'
 
+import MonitorLog from '../MonitorLog/MonitorLog'
```

Comment:
Could you add index.ts? to escape duplication 'MonitorLog' in the path

Reply:
done